### DISCUSSION TITLE
problem with ActiveSelection when holding SHIFT and selecting objects in interactive groups in v6

### DISCUSSION THREAD

**Discussion by thipm196** [fabricjs/fabric.js#8741] [General]
Active selection works strangely when holding SHIFT and selecting objects in interactive groups.
https://codesandbox.io/s/vigilant-raman-4m97dt?file=/src/App.vue:1243-1254
Here is my groups:
![image](https://user-images.githubusercontent.com/108201472/221413010-65238335-2ee9-4e66-af9c-8107c93d03fb.png)
When i holding SHIFT key and trying select blue and black rect in 2 different groups:
![image](https://user-images.githubusercontent.com/108201472/221413174-586362d2-8cb4-4eeb-872e-2e790bbd2c4e.png)

**Comment by ShaMan123** [comment]
can you try with current master?

**Reply by thipm196** [reply]
> can you try with current master?

Yah, this bug has been resolved in master. Could you tell me which merge request fixed it? I want to see what the issue was.

**Reply by ShaMan123** [reply]
#8665
